README.md

```diff
@@ -7,6 +7,10 @@
 * [dev](https://ruby-gnome.github.io/ruby-gnome/doc/dev/)
 * [latest](https://ruby-gnome.github.io/ruby-gnome/doc/latest/)
 
+## Open source
```

Comment:
```suggestion
## Source
```

If we want to emphasize "open source", we must describe our license too. Because open source is based on license.

Reply:
OK. How about "Source Code" ?

Reply:
Sounds good to me!

Reply:
The Apache Arrow project site links to GitHub from "Source Code".
![image](https://user-images.githubusercontent.com/5798442/187396538-c36f9fcc-b4b9-43bc-a990-faadde0139d5.png)